### DISCUSSION TITLE
Outline Kubernetes Helm Chart

### DISCUSSION THREAD

**Discussion by PatrickHuetter** [outline/outline#2636] [Show and tell]
Hey outline team,
thanks for your great work creating outline! We've always searched for some open source notion alternative and where stuck with nextcloud collectives. We weren't really glad with that because the nextcloud backends and especially collectives are very slow so we are glad that we found outline as an alternative.

Since our own infrastructure runs on kubernetes we created a kubernetes helm chart which makes it more easy to deploy outline in kubernetes environments. Highly configurable wether you want to ship with all dependencies (database, cache, s3) or only with some or without. 

Just wanted to know you where you can find the chart:
https://github.com/encircle360-oss/outline-helm-chart

Maybe it's worth a mention in the quickstart documents or readme as an alternative for docker native environments.

Best regards from germany
Patrick

**Comment by tommoor** [comment]
Thanks Patrick, appreciate you making this available to the community!

**Comment by Freundschaft** [comment]
thanks a lot, is there a repo for the chart available?

**Reply by almereyda** [reply]
The repository is linked in the OP:

- https://github.com/encircle360-oss/outline-helm-chart
- https://gitlab.com/encircle360-oss/helm-charts/outline

**Comment by almereyda** [comment]
Due to its search machine friendly title, I would like to use this conversation for a collection of preliminary work that can inform the effort here:

- #1168
- #1333
- #1976 
- #4080

and also
 
- https://wolveix.com/deploy-outline-wiki-on-kubernetes/

Given the repeating demand for a Kubernetes Helm chart, I would suggest the community organises around the encircl360 repository, and maintains it as an up-to-date source of truth for deploying Outline through Helm. This would include keeping it up to date and on par with upstream—right now it's still at 0.63.0.
